jms/src/test/scala/docs/scaladsl/JmsConnectorsSpec.scala

```diff
@@ -539,7 +539,7 @@ class JmsConnectorsSpec extends JmsSpec {
       val completionFuture: Future[Done] = Source(msgsIn).runWith(jmsSink)
       completionFuture.futureValue shouldBe Done
       // make sure connection was closed
-      connectionFactory.cachedConnection shouldBe 'closed
+      eventually { connectionFactory.cachedConnection shouldBe Symbol("closed") }
```

Comment:
[minor] Don't like the symbol shorthand?

Reply:
That's deprecated in Scala 2.13.